tina/config.ts

```diff
@@ -0,0 +1,58 @@
+import { defineConfig } from "tinacms";
+
+// Your hosting provider likely exposes this as an environment variable
+const branch =
+  process.env.GITHUB_BRANCH ||
+  process.env.VERCEL_GIT_COMMIT_REF ||
+  process.env.HEAD ||
+  "main";
+
+export default defineConfig({
+  branch,
+
+  // Get this from tina.io
+  clientId: process.env.NEXT_PUBLIC_TINA_CLIENT_ID,
+  // Get this from tina.io
+  token: process.env.TINA_TOKEN,
+
+  build: {
+    outputFolder: "admin",
+    publicFolder: "static",
+  },
+  media: {
+    tina: {
+      mediaRoot: "",
+      publicFolder: "static",
+    },
+  },
+  // See docs on content modeling for more info on how to setup new content models: https://tina.io/docs/schema/
+  schema: {
+    collections: [
+      {
+        name: "post",
+        label: "Posts",
+        path: "content/posts",
+        ui: {
+          router: ({ document }: { document: { _sys: { template: string; filename: string } } }) => {
```

Comment:
Shouldn't need to type out document 😲

Reply:
✅Fixed 

Looks like Copilot went overboard with this 😬